_commands/basics/df.md

```diff
@@ -0,0 +1,72 @@
+---
+---
+
+df
+--
+
+df displays the amount of disk space available on the file system containing each file name argument. If no file name is given, the space available on all currently mounted file systems is shown. Disk space is shown in 1K blocks by default, unless the environment variable POSIXLY_CORRECT is set, in which case 512-byte blocks are used.  
+
+~~~ bash
+$ df
+Filesystem     1K-blocks    Used Available Use% Mounted on
+udev             1004740       0   1004740   0% /dev
+tmpfs             204848   21672    183176  11% /run
+/dev/sda1       49702628 5077532  42077232  11% /
+...(Some more info)
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `df -h`
```

Comment:
I might even consider using this as the primary example... I always use `df -h`.